### PR TITLE
No more annoying popups! (if you have bionical gills, of course)

### LINKED ISSUE #13309: Membrane oxygenator CBM still gives drowning messages
Latest experimental (3577), going underwater with the membrane oxygenator CBM still displays drowning popups once you run out of 'normal' oxygen, though it takes power instead of health. This might be fine to simulate that you still have the feeling of drowning, but gameplay-wise it's extremely annoying to have to close the popup manually every tile you move - I would suggest either removing the message with the CBM and sufficient power, or at least just moving it to the normal text output instead of a seperate popup.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/25820377-membrane-oxygenator-cbm-still-gives-drowning-messages?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F146201&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by illi-kun**
Fixes #13309, namely this:

> but moving still gives the warning popup every single step you take, starting when (i assume) you run out of normal oxygen.

Sorry for adding absolutely unrelated translation fix to this PR, but it's so tiny for dedicated PR and so easy to do...

### CODE REVIEW

File: src/player.cpp [hunk 1]
```diff
@@ -7684,7 +7684,7 @@ void player::suffer()
 
     if(has_active_mutation("WINGS_INSECT")){
         //~Sound of buzzing Insect Wings
-        sounds::sound( pos(), 10, "BZZZZZ");
+        sounds::sound( pos(), 10, _("BZZZZZ"));
```
Comment (drbig) [inline src/player.cpp]: This one might benefit from `pgettext()` with a comment I suppose.

Comment (narc0tiq) [inline src/player.cpp]: It has a `//~`, which is a translator comment and gets grabbed by xgettext.

Comment (drbig) [inline src/player.cpp]: @narc0tiq Aye, good to know. The edit about plugging it into `sounds::sound()` though...

**Comment by illi-kun** [comment]
> actually does it even make sense? Translated ids(?) for the sound engine?

It's not an ID, it's onomatopoeic word, take a look:
bzzz

**Comment by drbig** [comment]
So `sounds::sound()` seems to take it and print it - at the least. But then this seems to be the only thing that would identify the sound to play - if so then `_()` shouldn't be applied at the time of printing?

**Comment by BevapDin** [comment]
@drbig `sounds::sound` has more parameters. This one is printed (and not used by the sound effects system), the next is the "is ambient sound?" boolean, the next is the sound effect id (not translated, used for playing a sound effect), the next is the sound effect variant (not translated, used for playing a sound effect).

**Pull request merged**